pkg/controller/plan/validation.go

```diff
@@ -125,6 +134,27 @@ func (r *Reconciler) validate(plan *api.Plan) error {
 	return nil
 }
 
+//
+// Validate the target namespace.
+func (r *Reconciler) validateTargetNamespace(plan *api.Plan) (err error) {
+	err = r.Get(context.TODO(), types.NamespacedName{Name: plan.Spec.TargetNamespace}, &v1.Namespace{})
+	if k8serr.IsNotFound(err) {
+		err = nil
+		newCnd := libcnd.Condition{
+			Type:     NamespaceNotValid,
+			Status:   True,
+			Category: Critical,
+			Reason:   NotFound,
```

Comment:
I think `NotSet` would be better.

Reply:
Maybe NotSet if it's empty, NotFound if the namespace doesn't exist?